acquamanCommon.pri

```diff
@@ -153,7 +153,15 @@ HEADERS += \
 	source/ui/dataman/AM2DScanView.h \
 	source/ui/AMDatamanStartupSplashScreen.h \
 	source/dataman/database/AMDbUpgrade.h \
+	source/dataman/databaseUpgrades/AMDbUpgrade1Pt1.h \
+	source/dataman/databaseUpgrades/AMDbUpgrade1Pt2.h \
+	source/dataman/databaseUpgrades/AMDbUpgrade1Pt3.h \
+	source/dataman/databaseUpgrades/AMDbUpgrade1Pt4.h \
+	source/dataman/databaseUpgrades/AMDbUpgrade1Pt5.h \
+	source/dataman/databaseUpgrades/AMDbUpgrade1Pt6.h \
+	source/dataman/CLS/CLSDbUpgrade1Pt1.h \
```

Comment:
Just curious, is there a semantic difference between AMDbUpgrades and CLSDbUpgrades?